crates/evm/src/executor/inspector/printer.rs

```diff
@@ -63,6 +63,6 @@ impl<DB: Database> Inspector<DB> for TracePrinter {
             hex::encode(&inputs.init_code),
             inputs.gas_limit
         );
-        (InstructionResult::Continue, None, Gas::new(0), Bytes::new())
+        (InstructionResult::Continue, None, Gas::new(0), alloy_primitives::Bytes(Bytes::new()))
```

Comment:
`Bytes` import can be replaced with `alloy_primitives::Bytes`. Also it has methods too, you don't have to wrap it everywhere

Reply:
Yep, done in https://github.com/foundry-rs/foundry/pull/5768/commits/91107449b7bbc8e19a70cdb9ffc9489732cb146a